packages/server/package.json

```diff
@@ -36,6 +36,7 @@
     "@packages/icons": "0.0.0-development",
     "ansi_up": "5.0.0",
     "ast-types": "0.13.3",
+    "base64url": "^3.0.1",
```

Comment:
Is it necessary to add a new dependency? AFAICT everything's running on the server, where we can use `Buffer.from(str, 'base64')` and `Buffer.toString('base64')` to decode/encode base64.

Reply:
I guess here it's used to convert between two base64 encoding schemes?

https://github.com/cypress-io/cypress/blob/6ca21d6a8755895302eb4fc285ced1664e720ed7/packages/server/test/unit/cloud/api_spec.js#L61

Reply:
Yeah, `base64url` is slightly different than `base64`. I think support for it landed in 14.16 but ran into issues bumping `@types/node` to not complain about it. The code for it [is pretty simple](https://github.com/brianloveswords/base64url/blob/fce104a75ebade9bbeb761d74153b46938c2fef4/src/base64url.ts#L14-L28), I had orignially inlined but then didn't think it was worth porting the unit tests for it vs just having it as a dependency. We can remove it later when we bump the `@types/node` and it's a proper encoding target